### PR TITLE
New help texts

### PR DESCRIPTION
**Pull request by manfredbrandl**
I give it a fresh try.

### CODE REVIEW

**Comment by manfredbrandl** [comment]
@bhousel please review. Some java-script is not perfect.

File: data/core.yaml [hunk 1]
```diff
@@ -547,282 +547,455 @@ en:
...
 
-      To select multiple features, hold down the 'Shift' key. Then either click
-      on the features you want to select, or drag on the map to draw a contour
-      around those features. All the points inside the lasso area will be selected.
+        The iD editor is a collaborative project with [source
+        code available on GitHub](https://github.com/openstreetmap/iD).
 
-      ### Saving Edits
+      othereditors: |
+        ### Other Editors
```
Comment (bhousel) [inline data/core.yaml]: 👎

**Comment by bhousel** [comment]
It overall looks really good!  I think what you did with the JavaScript is fine.  

Is this still work in progress or are you mostly finished?  When you are all done, I'd like to have a few more people look it over (especially inexperienced users) for ideas on things we could simplify.

**Comment by manfredbrandl** [comment]
@bhousel It is mostly finished, and of course I will remove the section about other editors. Please let a few people look on it, a native speaker would help, ideas for simplification or missing information, etc. would help now.

File: data/core.yaml (hunk 1 above)
Comment (manfredbrandl) [reply data/core.yaml]: I will remove this soon.

**Comment by manfredbrandl** [comment]
@bhousel I did some more polishing. Did anybody look over it?

**Comment by bhousel** [comment]
> I did some more polishing. Did anybody look over it?

Sorry @manfredbrandl not yet - I am trying to pull in a bunch of other things and do a release soon, then we will merge this one which will affect the translations significantly.  
 
(I've been giving priority to the PRs that are closing [Hacktoberfest](https://hacktoberfest.digitalocean.com/) issues, since for some people this is their credit for the promotion).

**Comment by manfredbrandl** [comment]
@bhousel no problem, the text needs a native speaker and a osm newbie to look over. i am happy to have that merged soon after the next release to give translators enough time till the following release.

**Comment by manfredbrandl** [comment]
@bhousel After the 2.5.0 release can you please find someone to review my texts?

**Comment by bhousel** [comment]
> After the 2.5.0 release can you please find someone to review my texts?

Yes, I'd like to merge it this week, just waiting a few days to make sure there are no bad bugs that require a 2.5.1.

**Comment by manfredbrandl** [comment]
@bhousel is now time for review or merge?

**Comment by bhousel** [comment]
> is now time for review or merge?

@manfredbrandl  yes! I started yesterday making my edits.  I probably would have merged it last night but I got tired.. basically:

- simplified the language, borrowing from the tutorial language wherever possible
- simplify the topics to "points","lines","areas", instead of "points & addresses", "lines & roads", "areas & buildings" (although I did keep in the mapping guidance about how to map addresses, roads, and buildings in those sections)
- added section on street level photos like Mapillary and OpenStreetCam
- removed the section that listed out the icons (mostly because all the icons already have hovertips, and I don't want to risk having this help section go stale) 

I still need to clean up the section on Relations, but I expect to merge it today and talk it up on Twitter to get people translating the content.  

Thank you again for all your work on this @manfredbrandl - the new help is sooo much better 🙇   Having it broken up into smaller sections will help translators a lot!

**Comment by manfredbrandl** [comment]
@bhousel I really appreciate having you improvements on my text. I hope to have enought time tommorow to translate to German, otherwise someone else will do it during my vacation.

**Comment by bhousel** [comment]
Moved to #4557... Excited to be close to merging this 🙌

**Pull request closed without merging**